scripts/utilities.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+
+import argparse
+import logging
+import os
+import sys
+
+from typing import List, Optional
+
+
+logger = logging.getLogger(__name__)
+
+
+DATABASES = ["panther", "ncbifam", "cathgene3d"]
+
+
+def clean_url(url):
+    return url.rstrip("/")
+
+
+def build_parser(argv: Optional[List] = None):
+    parser = argparse.ArgumentParser(
+        prog="Import AI generated data",
+        description="Automatically integrated AI-generated signatures into the IPPRO database",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "-b",
+        "--batch_size",
+        type=int,
+        default=50000000,
+        help="Number of signatures to parse at a time.",
+    )
+
+    parser.add_argument(
+        "-c",
+        "--cache",
+        action="store_false",
```

Comment:
With `action="store_false"`, the value of the argument (`args.cache`) is `True` by default and `False` if `-c` or `--cache` is passed on the command line.

However, if you add `default=False`, then `args.cache` will always be `False`.
I guess you want the cache file to be used if `-c` or `--cache` is passed on the command line so you need to use `action="store_true"`.

```suggestion
        action="store_true",
```